src/design-system.html

```diff
@@ -10,6 +10,25 @@
   </head>
   <body>
     <h1>Design System</h1>
-    <a href="#">VENHA CONHECER SUA PÁGINA AQUI</a>
+    <div>
+      <button class ="button button--general">STATIC</button>
```

Comment:
💅

```suggestion
<button class="button button--general">STATIC</button>
```

Mesma coisa para as outras linhas.